src/utils/scrape/login.ts

```diff
@@ -13,16 +13,17 @@ export const login = async (cookieJar: CookieJar) => {
   urlencoded.append("ctl00$ContentPlaceHolder1$Login1$Password", PASSWORD!);
   urlencoded.append("ctl00$ContentPlaceHolder1$Login1$LoginButton", "Zaloguj");
 
-  const requestOptions = {
+  const requestOptions: RequestInit = {
     method: 'POST',
     headers: myHeaders,
     body: urlencoded,
-    redirect: 'manual' as const
+    redirect: 'manual'
   };
 
   const firstResponse = await fetch("https://planzajec.pjwstk.edu.pl/Logowanie.aspx", requestOptions)
   const [firstCookie] = firstResponse.headers.raw()['set-cookie'][0].split(';')
-  const nextLocation = firstResponse.headers.get("location")!
+  const nextLocation = firstResponse.headers.get("location")
+  if (!nextLocation) throw new Error('Missing location header!')
```

Comment:
Just a styling preference

```suggestion
  const nextLocation = firstResponse.headers.get("location")

  if (!nextLocation) {
    throw new Error('Missing location header!')
  }
```

Reply:
I see curly barces gang {😎}